doc/over_react_redux_documentation.md

```diff
@@ -275,6 +279,270 @@ UiFactory<CounterProps> Counter = connect<CounterState, CounterProps>(
 
 > [More information about the `connect` function](https://react-redux.js.org/api/connect#connect)
 
+## Hooks
+OverReact offers a set of hook APIs as an alternative to the existing [`connect()`](#connect) Higher Order Component. 
+These APIs allow you to subscribe to the Redux store and dispatch actions, without having to wrap your components 
+in [`connect()`](#connect).
```

Comment:
We should clarify these APIs' relationship to React Redux and link to that documentation up front, similar to the `connect` section:

```suggestion
OverReact exposes wrappers around React Redux hook APIs, which serve as an alternative to the existing [`connect()`](#connect) Higher Order Component. 
These APIs allow you to subscribe to the Redux store and dispatch actions, without having to wrap your components 
in [`connect()`](#connect).

> See: https://react-redux.js.org/api/hooks
```